app/code/local/Dwyera/Pinpay/Model/Result.php

```diff
@@ -165,6 +165,32 @@ public function getResponseToken()
             throw new Dwyera_Pinpay_Model_ResponseParseException;
         }
     }
+    public function getCustomerToken()
+    {
+        if (isset($this->msgObj->response->token)) {
```

Comment:
@ramya-aligent  should we also check if response is set ?
May be create a function isResponseParamExists($param)
and it can throw exceptions and check issset